src/libraries/System.Net.WebSockets.WebSocketProtocol/ref/System.Net.WebSockets.WebSocketProtocol.cs

```diff
@@ -9,6 +9,6 @@ namespace System.Net.WebSockets
 {
     public static partial class WebSocketProtocol
     {
-        public static System.Net.WebSockets.WebSocket CreateFromStream(System.IO.Stream stream, bool isServer, string subProtocol, System.TimeSpan keepAliveInterval) { throw null; }
+        public static System.Net.WebSockets.WebSocket CreateFromStream(System.IO.Stream stream, bool isServer, string? subProtocol, System.TimeSpan keepAliveInterval) { throw null; }
```

Comment:
NIT: don't we want to add this annotation to the source to keep them same?

Reply:
Oops, yes, I neglected to commit that file.